src/ArchiveManagement/NexusMods.FileExtractor/Extractors/SevenZipExtractor.cs

```diff
@@ -221,9 +228,24 @@ private static string GetExtractorExecutable(IFileSystem fileSystem)
     /// See https://github.com/Nexus-Mods/NexusMods.App/issues/1306 for details.
     /// </remarks>
     private const bool UseSystemExtractor =
-#if USE_SYSTEM_EXTRACTOR
+#if NEXUSMODS_APP_USE_SYSTEM_EXTRACTOR || USE_SYSTEM_EXTRACTOR
         true;
 #else
         false;
 #endif
+    
+    private static string FindSystem7zOnLinux()
+    {
+        string[] potentialBinaryNames = ["7z", "7zz", "7zzs"];
+        var pathDirectories = Environment.GetEnvironmentVariable("PATH")!.Split(':');
```

Comment:
I was concerned at first that this could run into edge cases, but the POSIX spec specifically says directories included in PATH cannot contain the delimitator `:`.

In other words there's no escape sequences for `:` in PATH paths, so this simple implementation is the correct one.